### PR TITLE
Fix events and workflow_runs datetimes in `source-github`

### LINKED ISSUE #19298: `source-github` has many timestamps as strings that should really be "date-time"
In `source_github/schemas/events.json`
* `created_at` should gain `"format": "date-time"`

In `source_github/schemas/workflow_runs.json`
* `created_at` should gain `"format": "date-time"`
* `updated_at` should gain `"format": "date-time"`

### PR DESCRIPTION
**Pull request by evantahler**
closes https://github.com/airbytehq/airbyte/issues/19298

### CODE REVIEW

File: docs/integrations/sources/github.md [hunk 1]
```diff
@@ -209,8 +216,7 @@ The GitHub connector should not run into GitHub API limitations under normal usa
...
 | 0.1.4   | 2021-08-13 | [5136](https://github.com/airbytehq/airbyte/pull/5136)                                                            | Support syncing multiple repositories/organizations                                                                                                                 |
-| 0.1.3   | 2021-08-03 | [5156](https://github.com/airbytehq/airbyte/pull/5156)                                                            | Extended existing schemas with `users` property for certain streams                                                                                                 |
+| 0.1.3   | 2021-08-03 | [5156](https://github.com/airbytehq/airbyte/pull/5156)                                                            | Extended existing schemas with `users`property for certain streams                                                                                                  |
```
Comment (davydov-d) [inline docs/integrations/sources/github.md:219]: looks like a whitespace is missing here before `property`

File: docs/integrations/sources/github.md [hunk 2]
```diff
@@ -209,8 +216,7 @@ The GitHub connector should not run into GitHub API limitations under normal usa
...
 | 0.1.2   | 2021-07-13 | [4708](https://github.com/airbytehq/airbyte/pull/4708)                                                            | Fix bug with IssueEvents stream and add handling for rate limiting                                                                                                  |
-| 0.1.1   | 2021-07-07 | [4590](https://github.com/airbytehq/airbyte/pull/4590)                                                            | Fix schema in the `pull_request` stream                                                                                                                             |
```
Comment (davydov-d) [inline docs/integrations/sources/github.md:214]: and here after `the`

**Review by davydov-d — APPROVED** [review]

File: docs/integrations/sources/github.md (hunk 1 above)
Comment (evantahler) [reply docs/integrations/sources/github.md]: Thanks!

**Pull request merged**